collect_app/src/main/java/org/odk/collect/android/fragments/dialogs/ProgressDialogFragment.java

```diff
@@ -104,6 +106,11 @@ private void setupView(@NonNull AlertDialog dialog) {
             ((TextView) dialogView.findViewById(R.id.message)).setText(getArguments().getString(MESSAGE));
         }
 
+        if (getArguments() != null) {
+            dialog.setCancelable(getArguments().getBoolean(CANCELABLE));
```

Comment:
Did you have to set this on the dialog and the fragment? I'm wondering if we maybe only need it on one of them.

I find this really confusing with `DialogFragment` 😕. I don't understand why the `Fragment` and the `Dialog` both have ther dialog interface (like `setCancellable`)...

Reply:
Thanks @seadowg for pointing it out. We need it only on the `Fragment`. The `Dialog` is shown as part of the `Fragment`, so we need to be concerned about the `fragment` instead of the `dialog`.

Reply:
Ah cool. Nice find in the docs. That makes sense!